### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by anmeizhang** [marcelampc/d3net_depth_estimation#4]
I want to train the model to predict blur image depth, so I wonder  the loss function  when you train your model ?   thank you!

**Comment by marcelampc** [comment]
Hello! I use a simple L1, you may refer to this paper to know the reason: http://mcarvalho.ml/material/docs/2018/regression_losses_icip_2018.pdf

Compared to GAN and Eigen's loss, it's the simplest and still very effective!

Best

**Comment by anmeizhang** [comment]
I have saw this paper, but I  can't understand that why the final activation function is tanh ?   And if I used tanh , L1 loss can't work....

**Comment by anmeizhang** [comment]
My ground trueth are true depth values（from 0m to 10m), but the outputs of tanh are within [-1, 1]. So I want to ask what is your depth ground truth?  By the way, are your input images in the interval [0, 1] or [0,255]?   I am looking forward for your answers. Thank you !
